tests/unit/pandas_utils/test_lusid_pandas.py

```diff
@@ -303,3 +303,112 @@ def test_response_to_df_rename_property(self):
                 ]
             ).intersection(set(df_columns)),
         )
+
+    def test_list_of_lusid_objects(self):
+
+        test_instrument_response_1 = lusid.models.instrument.Instrument(
+            lusid_instrument_id="LUID_TEST1",
+            version=1,
+            name="Test LUID1",
+            identifiers={
+                "ClientInternal": "imd_35349922",
+                "Figi": "BBG000BRVH22",
+                "Isin": "GB00B1KJJ422",
+                "LusidInstrumentId": "LUID_ZJJ1IY22",
+                "Ticker": "WTB22",
+            },
+            state="Active",
+        )
+
+        test_instrument_response_2 = lusid.models.instrument.Instrument(
+            lusid_instrument_id="LUID_TEST2",
+            version=1,
+            name="Test LUID2",
+            identifiers={
+                "ClientInternal": "imd_35349900",
+                "Figi": "BBG000BRVH05",
+                "Isin": "GB00B1KJJ408",
+                "LusidInstrumentId": "LUID_ZJJ1IYJS",
+                "Ticker": "WTB",
+            },
+            state="Active",
+        )
```

Comment:
You may want to move the definition of these to the initialisation of the test and reference them via self to save duplicating them in multiple tests.